pages/apim/3.x/kubernetes/apim-kubernetes-operator-monitoring.adoc

```diff
@@ -0,0 +1,98 @@
+[[apim-kubernetes-operator-monitoring]]
+= Monitor the Gravitee Kubernetes Operator
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim-kubernetes-operator-monitoring.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+[label label-version]#BETA release#
+
+== Overview
+
+This section provides steps on how to monitor the Gravitee Kubernetes Operator (GKO) on an existing APIM-ready Kubernetes cluster.
+
+NOTE: If you do not have an existing Gravitee Kubernetes Operator, you can set up a new one - see the link:{{ '/apim/3.x/apim-kubernetes-operator-installation-cluster.html' | relative_url }} for details. Once you have the operator set up and running, return to this section to deploy the GKO.
+
+== Prerequisites
+
+Before you start the deployment process, ensure that you have access to the following libraries:
+
+- The kubectl command-line tool installed on your local machine, configured to connect to your cluster. You can read more about installing kubectl in the official documentation.
+
+- The Helm package manager installed on your local machine. You can read more about installing Helm in the link:https://helm.sh/docs/intro/install/[official documentation].
+
+== Deploying the Prometheus Stack
+
+The link:https://github.com/prometheus-operator/kube-prometheus[kube-prometheus repository] is a collection of Kubernetes manifests, link:https://grafana.com/[Grafana] dashboards, and link:https://prometheus.io/[Prometheus rules] combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
+
+=== STEP 1: Run the deployment script
+
+Run the following command in your command-line tool (the working directory does not matter) create the namespace which will be used to deploy the Prometheus stack:
+
+....
+kubectl create namespace monitoring
+kubectl config set-context --current --namespace=monitoring
+....
+
+Before installation, please check you’ve added the dedicated repository to helm:
```

Comment:
This part should be moved above the installation part above

Reply:
No I'm not sure sorry 😅. This is really logical to do it here according to the documentation I wrote and what the user will do. 
However I agree maybe we could rephrase this line. I proposed something else in my last push on the branch so feel free to tell me if it's better or if you really think we should move this section.

Reply:
It's perfectly clear now with the rewritten sub-step, thanks @Okhelifi :)